src/package.json

```diff
@@ -103,8 +104,10 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --timeout 120000 --recursive tests/backend/specs ../node_modules/ep_*/static/tests/backend/specs",
-    "test-container": "mocha --timeout 5000 tests/container/specs/api"
+    "test": "mocha --require ts-node/register --timeout 120000 --recursive tests/backend/specs ../node_modules/ep_*/static/tests/backend/specs",
+    "test-container": "mocha --require ts-node/register --timeout 5000 tests/container/specs/api",
+    "dev": "ts-node node/server.ts",
```

Comment:
you can run ts file whit `tsx` like that `node import="tsx" ./myScritp.ts`

Ref:
- [TSX](https://www.npmjs.com/package/tsx?activeTab=readme)
- [article about loader on nodejs](https://dev.to/_staticvoid/how-to-run-typescript-natively-in-nodejs-with-tsx-3a0c)

Reply:
Is tsx better than ts-node?

Reply:
with ts-node you call a script that runs the typescriopt script, whereas with tsx in loader node it's node that runs the script directly. Normally tsx is supposed to perform better, but I'm not 100% sure.